packages/server/lib/server-base.ts

```diff
@@ -87,7 +85,7 @@ const notSSE = (req, res) => {
 
 export class ServerBase<TSocket extends SocketE2E | SocketCt> {
   private _middleware
-  protected request: Request
+  protected request: any
```

Comment:
Removing `// no-ts-check` caused this to error out in weird ways. I am not really sure how to type this, `Request` is highly dynamic and get modified on the fly a lot - the full interface is not entirely clear, but `Request` does not seem to satisfy it.